src/main/java/com/minecolonies/coremod/entity/ai/basic/AbstractEntityAIStructureWithWorkOrder.java

```diff
@@ -95,7 +98,7 @@ public IAIState loadRequirements()
         if (!job.hasBlueprint() || structurePlacer == null)
         {
             loadStructure();
-            final WorkOrderBuildDecoration wo = job.getWorkOrder();
+            final AbstractWorkOrder wo = job.getWorkOrder();
```

Comment:
should probably use the interface not the abstract type here, at other places aswell